tasks/webhook

```diff
@@ -100,17 +100,21 @@ class ReleaseHandler(github_handler.GithubHandler):
         title = request['pull_request']['title']
         number = int(request['number'])
         action = request['action']
+        merged = request['pull_request']['merged']
```

Comment:
Not sure if that's guaranteed to exist? The common approach is `.get('merged', False)`, but maybe like that it's better initially, so that it doesn't always silently default to False even when we expect it to exist.